zerver/webhooks/travis/tests.py

```diff
@@ -114,25 +114,6 @@ def test_travis_exclude_glob_events(self) -> None:
             expect_noop=True,
         )
 
-    def test_travis_invalid_event(self) -> None:
-        payload = self.get_body("build")
-        payload = payload.replace("push", "invalid_event")
-        expected_error_messsage = """
-Error: This test triggered a message using the event "invalid_event", which was not properly
-registered via the @webhook_view(..., event_types=[...]). These registrations are important for Zulip
-self-documenting the supported event types for this integration.
-
-You can fix this by adding "invalid_event" to ALL_EVENT_TYPES for this webhook.
-""".strip()
```

Comment:
(This is was a silly test, as it checked for an error message that was generated by the test suite itself.)

https://github.com/zulip/zulip/blob/160a8b99459d3f4073444b906dec35e67e1848ba/zerver/lib/test_classes.py#L1635-L1643